modules/blocks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Load code specific to Gutenberg blocks which are not tied to a module.
+ * This file is unusual, and is not an actual `module` as such.
+ * It is included in ./module-extras.php
+ *
+ */
+
+jetpack_register_block(
```

Comment:
Could you add docblocks to all the functions you add here? I suspect this file is going to get bigger and bigger, so it's probably best if we document everything that is going to go there from the start.

Reply:
Sure thing. What should I be using for the `@source` tag?

Reply:
I would suggest sticking to a description, `@param`, and `@return` tags for now. That should be enough to help people use those functions and get IDE suggestions. What do you think?

Reply:
Added the docblock here: https://github.com/Automattic/jetpack/pull/10503/commits/076cb38f8f01a4070156bb99b8676c24d2afef40